WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerFragment.kt

```diff
@@ -116,4 +118,7 @@ class HomePagePickerFragment : Fragment() {
             AniUtils.fadeOut(title, Duration.SHORT, View.INVISIBLE)
         }
     }
+
+    private fun isPhoneLandscape() = DisplayUtils.isLandscape(requireContext()) &&
```

Comment:
Can we use [this](https://github.com/wordpress-mobile/WordPress-Android/pull/13218/files#diff-c21c814a771a8de931459e462bb91a30bd9224cbe1488f5027e231f17d483da7R13) here?

Reply:
Good idea. Fixed with cda4fac3a662a4b863367ff6aa0778dc7a7f0b9c

Reply:
Nice!